src/app.ts

```diff
@@ -1,6 +1,89 @@
+type ResultData = {
+  id: string;
+  gender: string;
+  age: number;
+  row: number;
+};
+
+type PagingData = {
+  next?: string;
+  previous?: string;
+};
+
+let previousBtn: HTMLElement | any = document.querySelector("button[data-prevbtn]");
```

Comment:
Codacy has a fix for the issue: [Replace `"button[data-prevbtn]"` with `⏎··"button[data-prevbtn]"⏎`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
let previousBtn: HTMLElement | any = document.querySelector(
```